specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/codeContainer.json

```diff
@@ -0,0 +1,481 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "Azure Machine Learning Workspaces",
+		"description": "These APIs allow end users to operate on Azure Machine Learning Workspace resources.",
+		"version": "2020-09-01-preview"
+	},
+	"host": "management.azure.com",
+	"schemes": [
+		"https"
+	],
+	"consumes": [
+		"application/json"
+	],
+	"produces": [
+		"application/json"
+	],
+	"paths": {
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/Codes/codeversions": {
```

Comment:
TThis should end with "/codes". You can remove the "codeversions" part (bug).

Reply:
should I lowercase all `/Codes` to `/codes`?